CMakeLists.txt

```diff
@@ -541,6 +541,7 @@ add_subdirectory(test/includes)
 add_subdirectory(config)
 add_subdirectory(test/functional/fixtures)  # compile test programs
 add_subdirectory(runtime)
+get_directory_property(GENERATED_HELP_TAGS DIRECTORY runtime DEFINITION GENERATED_HELP_TAGS)
```

Comment:
is this redefining `GENERATED_HELP_TAGS`? should it use a different var name, e.g. `GENERATED_HELP_TAGS_DIR`

Reply:
It is pulling it in from the subdirectory.
Could have a different name, but it is the same after all.